examples/intel-optimized-mysql-server-replica_testing/main.tf

```diff
@@ -5,23 +5,22 @@
 
 module "optimized-mysql-server" {
   source         = "intel/aws-mysql/intel"
-  rds_identifier = "mysql-dev"
+  rds_identifier = "mysqldev"
   db_password    = var.db_password
 
   # Update the vpc_id below for the VPC that this module will use. Find the default vpc-id in your AWS account
   # from the AWS console or using CLI commands. In your AWS account, the vpc-id is represented as "vpc-",
   # followed by a set of alphanumeric characters. One sample representation of a vpc-id is vpc-0a6734z932p20c2m4
-  vpc_id = "<ENTER VPC_ID HERE>"
+  vpc_id = "vpc-0abf36f0171c55ae1"
 }
 
 module "optimized-mysql-server-read-replica" {
   source         = "intel/aws-mysql/intel"
-  rds_identifier = "mysql-dev-replica"
+  rds_identifier = "mysqldevreplica"
   db_password    = var.db_password
 
   # Update the vpc-id below. Use the same vpc-id as the one used in the prior module.
-  vpc_id                           = "<ENTER VPC_ID HERE>"
-  aws_database_instance_identifier = "mysql-rr"
+  vpc_id                           = "vpc-0abf36f0171c55ae1"
```

Comment:
Do we want to use `var.vpc_id` instead?

Reply:
same on here..  vpc_id?

Reply:
Changed to <ENTER VPC_ID HERE>
We'll make it a variable in the future.